src/styles/styles.js

```diff
@@ -798,6 +798,50 @@ const styles = {
         justifyContent: 'center',
     },
 
+    emojiPickerContainer: {
+        backgroundColor: themeColors.componentBG,
+        minWidth: CONST.EMOJI_PICKER_SIZE,
+    },
+
+    emojiPickerList: {
+        height: 300,
+        width: '100%',
+        paddingHorizontal: 16,
+    },
+
+    emojiHeaderStyle: {
+        backgroundColor: themeColors.componentBG,
+        width: '100%',
+        paddingVertical: 12,
+        fontFamily: fontFamily.GTA_BOLD,
+        fontWeight: fontWeightBold,
+        color: themeColors.heading,
+        fontSize: variables.fontSizeSmall,
+    },
+
+    // Emoji Picker Styles
+    emojiText: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: variables.iconSizeLarge,
+        paddingTop: 4,
+        paddingBottom: 4,
+        paddingRight: 8,
+        paddingLeft: 8,
+    },
+
+    emojiItem: {
+        width: '12.5%',
+        textAlign: 'center',
+    },
+
+    chatItemEmojiButton: {
+        alignSelf: 'flex-end',
+        borderRadius: 6,
+        height: 32,
+        margin: 3,
```

Comment:
`m1` - I think margins should be in units of 4 ?

Reply:
I think with the nested button within the compose input, we are making a bit of an exception. This is basically using the same styles as the send button.

Reply:
Ok, well you wrote the rule so you can break em

Reply:
Just out here trying to enforce for ya